onward/app/feed/MostViewedGalleryAgent.scala

```diff
@@ -32,7 +27,7 @@ object MostViewedGalleryAgent extends Logging with ExecutionContexts {
         url <- (item \ "url").asOpt[String]
         count <- (item \ "count").asOpt[Int]
       } yield {
-        getResponse(LiveContentApi.item(UrlToContentPath(url), Edition.defaultEdition)).map(_.content.map(Content(_)))
```

Comment:
This makes 1 CAPI call per piece of content.

That can be dropped to 1 call in total if we do an ID search instead.